q_types.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python3
+
+"""
+Class for each question type available on Blackboard. Fill-in question
+types are not included
+"""
+import re
+
+class Q_process:
+    """Defualt initialisation for all question handling classes
+    
+    questions (dict) -> assigns generic values to object
+    """
+
+    def __init__(self, question):
+        self.question = question
+        self.type = question['type']
+        self.prompt = question['prompt']
+
+class MC(Q_process):
```

Comment:
Can you add a docstring at the top of this module explaining what the classes are and the general idea of the code and linking to a source that explains the questions types and what the abbreviations "MC" etc mean